lib/compress/inline.js

```diff
@@ -148,12 +150,46 @@ function scope_encloses_variables_in_this_scope(scope, pulled_scope) {
     return false;
 }
 
+/**
+ * if definition is const or class, check reference length of definition is longer or
+ * short than init const value. for example
+ * ```js
+ * // top_retain: ["example"]
+ * const example = 100;
+ * console.log(example);
+ * ``` 
+ * length of init value is longer then identifier, it would return true.
+ * for non const or class AST node, always return true.
+ * @param {*} def 
+ * @return {bool}
+ */
+function check_const_symbol_and_value_length(def) {
+    if(
+        (def.orig.length === 1 && def.orig[0] instanceof AST_SymbolConst) ||
+        (def.orig.length === 1 && def.orig[0] instanceof AST_SymbolDefClass)
+    ) {
+       if(def.init || def.orig[0].init) {
+        const init = def.init || def.orig[0].init;
+        const init_value_length = init.start === init.end ? String(init.value).length :(init.end.pos - init.start.pos + 1);
```

Comment:
```suggestion
        const init_value_length = init.size();
```

The size method should do it